tests/python_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,31 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`comments.py 1`] = `
+if a:  # a
+    # b
+    c
+
+    # c
+
+    d
+
+    # e
+
+    call(
+        # a
+    )
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+if a: # a
```

Comment:
This isn't quite correct. PEP 8 calls for two spaces before inline comments.

Reply:
This is an odd requirement :)

Reply:
See https://www.python.org/dev/peps/pep-0008/#inline-comments

> Inline comments should be separated by at least two spaces from the statement.

Yes, it's sort of weird, but it's more or less universal in Python-land.